homework/index.js

```diff
@@ -27,21 +30,138 @@
         elem.setAttribute(key, value);
       }
     });
+
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  // fetch url contributor
+  function fetchContributorsUrl(contributor) {
+    const cont = document.getElementById('cont');
+    cont.innerHTML = '';
+    fetchJSON(contributor)
+      .then(myContributor => {
+        for (let j = 0; j < myContributor.length; j++) {
+          const nameOfContributor = myContributor[j].login;
+          const numberOfContributor = myContributor[j].contributions;
+          const avatarUrl = myContributor[j].avatar_url;
+          const avatarContainer1 = createAndAppend('div', cont, {
+            class: 'avatar_div',
+          });
+          const avatarContainer2 = createAndAppend('div', cont, { class: 'avatar_div data' });
+
+          createAndAppend('img', avatarContainer1, { src: avatarUrl });
+          createAndAppend('a', avatarContainer2, {
+            text: nameOfContributor.toUpperCase(),
+            href: `https://github.com/${nameOfContributor}`,
+          });
+          createAndAppend('br', avatarContainer2, {});
+
+          createAndAppend('span', avatarContainer2, {
+            text: 'Forks: ',
+          });
+          createAndAppend('span', avatarContainer2, {
+            text: numberOfContributor,
+            class: 'result label',
+          });
+          createAndAppend('br', avatarContainer2, {});
+        }
+      })
+      .catch(() => {
+        createAndAppend('div', cont, {
+          text: 'there is no contributors available',
+          class: 'error',
+        });
+      });
+  }
+
+  function reloadSelectedRepository(valueOfSelectedRepository, selectedData) {
+    const info = document.getElementById('info');
+    info.innerHTML = '';
+    const getDescription = selectedData[valueOfSelectedRepository].description;
+    const getForks = selectedData[valueOfSelectedRepository].forks;
+    const getUpdatedAt = selectedData[valueOfSelectedRepository].updated_at;
+    const dataName = selectedData[valueOfSelectedRepository].name;
+    const getContributorsUrl = selectedData[valueOfSelectedRepository].contributors_url;
+
```

Comment:
The `get` prefix in variable name indicates this variable is function, but it is not the case here, variables should be nouns in general.